### PR TITLE
Verify identity button reordering

### PR DESCRIPTION
**Pull request by relaxnow**
https://www.pivotaltracker.com/n/projects/1163646/stories/100299518 points 10, 11 and 12

### CODE REVIEW

**Comment by relaxnow** [comment]
Depends on https://github.com/SURFnet/Stepup-RA/pull/111

**Comment by relaxnow** [comment]
screen shot 2016-06-14 at 14 59 35

**Comment by relaxnow** [comment]
@rjkip thoughs? suggestions? poems? songs? angry rants?

**Comment by rjkip** [comment]
> Hello, mate,
> These UI changes look great

:Ok_hand:

**Comment by rjkip** [comment]
:(

**Comment by arothuis** [comment]
Looks fine to me :+1:

File: src/Surfnet/StepupRa/RaBundle/Controller/Vetting/SmsController.php [hunk 1]
```diff
@@ -112,6 +113,12 @@ public function provePossessionAction(Request $request, $procedureId)
             ->createForm('ra_verify_phone_number', $command, ['procedureId' => $procedureId])
             ->handleRequest($request);
 
+        /** @var SubmitButton $cancelButton */
+        $cancelButton = $form->get('cancel');
+        if ($cancelButton->isClicked()) {
+            return $this->forward('SurfnetStepupRaRaBundle:Vetting:cancelProcedure', ['procedureId' => $procedureId]);
```
Comment (rjkip) [inline src/Surfnet/StepupRa/RaBundle/Controller/Vetting/SmsController.php]: Could you perform the same change here?

File: src/Surfnet/StepupRa/RaBundle/Resources/views/Vetting/Sms/provePossession.html.twig [hunk 2]
```diff
@@ -20,10 +20,36 @@
...
+    <div class="row">&nbsp;</div>
+    <div class="row offset-top">
+        <div class="col-md-1">
+            &nbsp;
+        </div>
+        <div class="col-md-5">
+            {{ form_row(form.resendChallenge) }}
+        </div>
+        <div class="col-md-3">
+            <button name="ra_verify_phone_number[cancel]" class="btn btn-danger pull-right" formnovalidate="formnovalidate">
```
Comment (rjkip) [inline src/Surfnet/StepupRa/RaBundle/Resources/views/Vetting/Sms/provePossession.html.twig]: And here :sweat_smile:

File: src/Surfnet/StepupRa/RaBundle/Controller/Vetting/SmsController.php (hunk 1 above)
Comment (relaxnow) [inline src/Surfnet/StepupRa/RaBundle/Controller/Vetting/SmsController.php]: No need, the merge has fixed this. I don't get why GitHub doesn't pick this up?

Comment (rjkip) [inline src/Surfnet/StepupRa/RaBundle/Controller/Vetting/SmsController.php]: Okay, great. No worries :)

**Pull request merged**